monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -482,6 +482,20 @@ abstract class Observable[+A] extends Serializable { self =>
   final def bufferIntrospective(maxSize: Int): Observable[List[A]] =
     new BufferIntrospectiveObservable[A](self, maxSize)
 
+  /** Implementation of `bracket` from `cats.effect.Bracket`.
+    *
+    * See [[https://typelevel.org/cats-effect/typeclasses/bracket.html documentation]].
```

Comment:
I've followed `Iterant`'s doc but I was thinking about copying scaladoc from `cats.effect.Bracket` - is it ok/legal? I think it might be easier for users just browsing it in IDE and wondering what it does